internal/trust/remotes.go

```diff
@@ -73,7 +73,7 @@ func (r *Remotes) Load(dir string) error {
 		remoteData[remote.Name] = *remote
 	}
 
-	if len(remoteData) == 0 {
+	if len(remoteData) == 0 && len(r.data) != 0 {
```

Comment:
In which situation would this condition be true? I understand in case of init `remoteData` is empty but what would be the effect of also checking `len(r.data) != 0`? Is it checking for the existence of the nodes own certificate? 
I would appreciate a comment here :)

Reply:
We try to load the truststore as soon as the daemon starts, but it won't contain anything until the node is bootstrapped. 

This results in a bunch of logs being sent because the truststore was initialized/refreshed, but without any content (because there never was any to begin with).

So this instead makes sure that the truststore actually had content before, and only propagates the log and abort if the truststore refresh causes all entries to be lost. 

I'll add a comment too.

Reply:
Thanks clear now.